src/windows/footpath.c

```diff
@@ -704,6 +705,48 @@ static void window_footpath_set_provisional_path_at_point(int x, int y)
 }
 
 /**
+*
+*  rct2: 0x006A8388
+*/
+static void window_footpath_set_selection_start_bridge_at_point(int screenX, int screenY)
+{
+	int x, y, direction;
+	rct_map_element *mapElement;
+
+	map_invalidate_selection_rect();
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_FLAGS, uint16) &= ~(1 << 0) & (1 << 2);
+
+	sub_68A0C9(screenX, screenY, &x, &y, &direction, &mapElement);
+	if (x == 0x8000)
+		return;
+
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_FLAGS, uint16) |= (1 << 0) | (1 << 2);
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_TYPE, uint16) = 4;
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_A_X, uint16) = x;
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_B_X, uint16) = x;
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_A_Y, uint16) = y;
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_SELECTION_B_Y, uint16) = y;
+
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_ARROW_DIRECTION, uint8) = direction;
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_ARROW_X, uint16) = x;
+	RCT2_GLOBAL(RCT2_ADDRESS_MAP_ARROW_Y, uint16) = y;
+
+	int z = mapElement->base_height;
+
+	if ((mapElement->type & 0x3C) == 0) {
```

Comment:
I think for most of the code we now use `map_element_get_type(type) == MAP_ELEMENT_TYPE_SURFACE` for that sort of comparison.